insights/specs/datasources/ssl_certificate.py

```diff
@@ -1,16 +1,25 @@
 """
 Custom datasource to get ssl certificate file path.
 """
-from insights.combiners.httpd_conf import HttpdConfTree
+import os
+
 from insights.combiners.nginx_conf import NginxConfTree
 from insights.core.context import HostContext
 from insights.core.exceptions import SkipComponent
 from insights.core.plugins import datasource
 from insights.parsers.mssql_conf import MsSQLConf
 from insights.combiners.rsyslog_confs import RsyslogAllConf
+from insights.specs import Specs
+from insights.specs.datasources import httpd
+
+
+class LocalSpecs(Specs):
+    """ Local specs used by httpd_certificate_info_in_nss and httpd_ssl_certificate_files datasource """
+
+    httpd_files = httpd.httpd_configuration_files
 
 
-@datasource(HttpdConfTree, HostContext)
+@datasource(LocalSpecs.httpd_files, HostContext)
```

Comment:
`LocalSpecs` is not so required, you can go ahead and use the `httpd.httpd_ssl_certificate_files`.